scripts/script.yaml

```diff
@@ -949,28 +973,38 @@
 
     - name: exposures
       steps:
-        - say: "חשוב לנו לדעת -"
+        - say: "הרבה אנשים נפגשים עכשיו, בעבודה ובמקומות אחרים -"
         - say: "האם ביממה האחרונה שהית בקרבת אנשים (<b>שאינם בני הבית</b>) למשך יותר מ-15 דקות ובמרחק של פחות מ-2 מטרים"
         - wait:
+            variable: public_place
             options:
-              - show: כן, שהיתי בקרבת אנשים נוספים
+              - show: כן, שהיתי בקרבת אנשים נוספים יותר מרבע שעה
                 steps:
-                - say: כמה מהם מבוגרים מעל גיל 18?
-                - wait:
-                    variable: _met_above_18
-                    placeholder: מספר המבוגרים, 0-99
-                    validation: "[0-9]+"
-                    input-kind: number
-                    input-min: 0
-                    input-max: 99
-                - say: וכמה ילדים מתחת לגיל 18?
+                - say: האם זה היה במקום ציבורי גדול?
+                - say: ... למשל, מקום עבודה, בית ספר או מרכז מסחרי?
                 - wait:
-                    variable: _met_under_18
-                    placeholder: מספר הילדים, 0-99
-                    validation: "[0-9]+"
-                    input-kind: number
-                    input-min: 0
-                    input-max: 99
+                    options:
+                    - show: כן
+                      value: true 
```

Comment:
ויצמן ביקשו את האבחנה הזו?

Reply:
לא, זה ניסיון שלי להתמודד עם השאלות שהתחלנו לקבל בשבועיים האחרונים - וחוסר היכולת לתת מספר אנשים ששהיתי איתם בכיתה/בית הספר/עבודה. ישלח אליהם לאישור ומשוב, כמובן.

Reply:
see the recent changes following #331.